functions-and-scope.js

```diff
@@ -72,6 +149,26 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 9
 
+//-----------------PLAN VAN AANPAK-----------------//
+//[x] Vervolgens moet een variabele met highest grade gemaakt worden (op 0)
+//[x] Als eerste moet er door de array geloopt worden om te kijken wat de cijfers zijn
+//[x] Wanneer er een cijfer wordt gevonden dat hoger is dan moet hij die waarde overschrijven
+//[x] Na de loop moet deze variabele teruggegeven worden
+
+console.log("Opdracht 3a:");
+console.log("----------------------");
+
+let highestGrade = 0;
```

Comment:
Het is geen goed voorbeeld, maar ik wil het je wel meegeven:
Wanneer je een array meekrijgt ([-1, -3, -2]) met alleen maar negatieve getallen, dan zal de functie altijd 0 teruggeven. Het is daarom verstandig om altijd de eerste waarde in de array te pakken:
```suggestion
let highestGrade = grades[0];
```